src/test/java/uk/gov/hmcts/ccd/domain/service/common/AttributeBasedSecurityClassificationServiceTest.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.hmcts.ccd.domain.service.common;
+
+import com.google.common.collect.Lists;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import uk.gov.hmcts.ccd.data.casedetails.SecurityClassification;
+import uk.gov.hmcts.ccd.data.user.UserRepository;
+import uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol.AccessProfile;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
+import uk.gov.hmcts.ccd.domain.service.casedataaccesscontrol.CaseDataAccessControl;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class AttributeBasedSecurityClassificationServiceTest {
```

Comment:
Maybe worth adding a comment to `SecurityClassificationServiceTest` to remind whoever ends up removing that in the future to move the tests into here

Reply:
added test cases related to accesscontrol logic